x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/manual_configuration.tsx

```diff
@@ -59,6 +173,33 @@ export const ManualConfiguration: React.FC<ManualConfigurationProps> = ({
         { defaultMessage: 'Manual configuration' }
       )}
     </EuiContextMenuItem>,
+    <EuiContextMenuItem
+      key="edit"
+      icon="console"
+      onClick={() => {
+        closePopover();
+      }}
+      css={css`
```

Comment:
do we need the custom css here? "Try with CLI" option doesn't have it

Reply:
We do, passing this CSS to the TryInConsoleButton component doesn't get this prop. We might improve this component to receive also CSS modifications like this one needed to be rendered exactly as a Popover option. cc @TattdCodeMonkey